packages/dev/gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -47,6 +47,9 @@ export class AdvancedDynamicTexture extends DynamicTexture {
     /** Snippet ID if the content was created from the snippet server */
     public snippetId: string;
 
+    /** Observable that fires when the GUI is ready */
+    public onGuiReadyObservable = new Observable<AdvancedDynamicTexture>();

```

Comment:
Can we clear in dispose just to be safe if we never reach ready before dispose ???